backend/curriculum_tracking/tests/test_api_views.py

```diff
@@ -235,6 +246,47 @@ def test_set_project_link(self):
         self.assertEqual(project.link_submission, link_2)
 
 
+class TestLongRunningActorsCalledFromAgileCardAPIView(APITestCase, APITestCaseMixin):
```

Comment:
Which long running actor specifically? There are a few. Can we be more explicit?